src/wallet/rpcwallet.cpp

```diff
@@ -4420,6 +4454,9 @@ size_t EstimateTxSize(
     }
     txsize += CTXOUT_REGULAR_SIZE * taddrRecipientCount;
 
+    if (orchardRecipientCount) {
+        txsize += (ZC_ZIP225_BASE_SIZE + ZC_ZIP225_VARIABLE_SIZE * std::max(2, (int) orchardRecipientCount) - 1);
```

Comment:
I don't understand `std::max` here; should it be `std::min`? Why is either is needed; I would think `orchardRecipientCount - 1` (I'm assuming the `BASE` variables include one orchard recipient, hence the `- 1`)?. In any event, a comment here would be helpful.

Nit (feel free to ignore): the outer parenthesis aren't needed; it's an idiom in C++ to omit them in this kind of "increment" statement (we write `x += expression` rather than `x += (expression)` no matter how complex `expression` is).

Reply:
```suggestion
        // - The Orchard transaction builder pads to a minimum of 2 actions.
        // - We subtract 1 because `GetSerializeSize(tx, ...)` already counts a
        //   single byte for the empty Orchard component.
        txsize += (ZC_ZIP225_BASE_SIZE + ZC_ZIP225_VARIABLE_SIZE * std::max(2, (int) orchardRecipientCount) - 1);
```

Reply:
Force-pushed to address review comments